Tests/SwerverTests/RouterSpec/RouterSpec.swift

```diff
@@ -9,10 +9,27 @@ class RouterSpec: QuickSpec {
             var router = Router()
```

Comment:
This can be part of a different refactoring, but I would avoid creating objects outside the scope of a test. The danger here is that the same instance will persist from test to test, resulting in the effects of one test persisting to the next test run. This can cause issues that are very hard to debug.

In general, tests should be completely isolated from each other, and a given test should not depend on another test or tests.

This change is definitely out of scope for this pull request.

Reply:
ok

Reply:
I would consider how to keep track of all the comments that are "pushed" to future commits. It might be something as simple as creating cards in Trello and shoving them in the backlog.

Reply:
will track those using trello cards